app/components/card-expense.tsx

```diff
@@ -34,21 +34,23 @@ export function CardExpense({
         <ul>
           {expenses.map((expense) => (
             <li className="my-2" key={expense.transaction_code}>
-              <div className="flex justify-between">
-                <h3 className="text-lg font-medium">{expense.description}</h3>
-                <h3 className="text-lg font-medium">
-                  {new Intl.NumberFormat("id-ID", {
-                    style: "currency",
-                    currency: "IDR",
-                  }).format(expense.idr_amount.toNumber())}
-                </h3>
-              </div>
+              <a href={`/expenses/123`}>
+                <div className="flex justify-between">
+                  <h3 className="text-lg font-medium">{expense.description}</h3>
+                  <h3 className="text-lg font-medium">
+                    {new Intl.NumberFormat("id-ID", {
+                      style: "currency",
+                      currency: "IDR",
```

Comment:
The currency won't always be `IDR` though

Reply:
Hmm, what should I use then? Should I use original_currency or do we have an env for deciding which currency to show?

Reply:
You should show both original currency and IDR currency though. The original transaction won't always be in IDR, but people who see this site mostly will come from Indonesia. So there's that.